lib/ex_doc/formatter/html.ex

```diff
@@ -119,6 +120,20 @@ defmodule ExDoc.Formatter.HTML do
     [sidebar_items]
   end
 
+  defp generate_contents_json(nodes_map, extras, config) do
```

Comment:
We should probably unify the `sidebar_items` with the `contents_json`. What do you think? But this can also be done in a separate pull request, I thought I would start the discussion though. :)

Reply:
Right, that makes sense, since I've had to make many things as in ```sidebar_items```